src/helpers/FaviconsOpenClimbing.tsx

```diff
@@ -0,0 +1,139 @@
+// this file is ignored by prettier
+import React from 'react';
+
+/*
+
+// ios - pwa maskable icons (white background)
+57x57 (iOS)
+60x60 (iOS)
+72x72 (iOS)
+76x76 (iOS)
+114x114 (iOS)
+120x120 (iOS)
+152x152 (iOS)
+180x180 (iOS)
+
+// others - pwa maskable icons (transparent)
+48x48
+96x96
+144x144
+192x192
+256x256
+384x384
+512x512
+
+*/
+
+export const FaviconsOpenClimbing = () => (
+  <>
+    <link
+      rel="shortcut icon"
+      href="/openclimbing/logo/osmapp_192.png"
+      sizes="192x192"
+    />
+    <link
+      rel="shortcut icon"
+      href="/openclimbing/logo/osmapp_256.png"
+      sizes="256x256"
+    />
+    <link rel="shortcut icon" href="/openclimbing/logo/osmapp_64.png" />
+    <link
+      rel="shortcut icon"
+      href="/openclimbing/favicon.ico"
+      type="image/x-icon"
+    />
+    {/* Generated by https://realfavicongenerator.net/ */}
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/openclimbing/logo/fav/apple-touch-icon.png"
+    />{' '}
+    {/* apple-touch-icon must have padding + white background */}
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/openclimbing/logo/fav/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/openclimbing/logo/fav/favicon-16x16.png"
+    />
+    <link
+      rel="mask-icon"
+      href="/openclimbing/logo/fav/safari-pinned-tab.svg"
+      color="#eb5757"
+    />{' '}
+    {/* For safari tab-favicon */}
+    <link rel="manifest" href="/manifest-osmapp.json" />
+    <meta name="application-name" content="OsmAPP maps" />
+    <meta name="apple-mobile-web-app-title" content="OsmAPP maps" />
+    <meta name="description" content="A universal OpenStreetMap app" />
```

Comment:
Could you please remove this line in both files? It is already covered properly in `<OpenGraphTags>` component